firstproject/widget/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Paris</title>
+    <link rel="stylesheet" href="../css/styles.css">
+    <link rel="stylesheet" href="../font/lato_regular.ttf">
+</head>
+<style>
+
+</style>
+
+<body>
+  <div class="main_container">
+    <div class="widget_container">
+
+        <div class="date_temperature_block">
+          <div class="date">
+            <span class="current_day">Friday </span>
+            <div>
+              <span class="current_month">August,</span>
+              <span class="current_date">23rd</span>
+             </div>
+          </div>
+          <div class="temperature">
+            <span class="twelve"> 12°</span>
+            <span class="city"> Paris,France </span>
+            <div>
+              <span class="icon_wind"><img src="../icons/wind.png" alt="wind"></span>
+              <span class="speed">14</span>
+              <span class="mph"> mph</span>
+            </div>
+          </div>
+        </div>
+
+        <div class="bottom_week_arrows_block">
+            <div class="prev">
+             <div> &#10094; </div> <!--maybe it's better to use an icon arrow-->
+            </div>
+
+          <div class="days">
```

Comment:
Maybe it's better to display days via `<ul>` and `<li>` to avoid too much elements with `div` selector.